File: tests/fixtures/negative/security/StaticIv.java
package test;

import java.security.SecureRandom;

class Crypto {
    private byte[] field = new byte[16];
    void params(byte[] param) {
        byte[] random = new byte[16];
        new SecureRandom().nextBytes(random);
        new javax.crypto.spec.IvParameterSpec(param);
        new javax.crypto.spec.IvParameterSpec(field);
        new javax.crypto.spec.IvParameterSpec(random);
    }
}

File: tests/fixtures/negative/security/StaticIv.kt
package test

import java.security.SecureRandom

class Crypto {
    private val field = ByteArray(16)
    fun params(param: ByteArray) {
        val random = ByteArray(16)
        SecureRandom().nextBytes(random)
        javax.crypto.spec.IvParameterSpec(param)
        javax.crypto.spec.IvParameterSpec(field)
        javax.crypto.spec.IvParameterSpec(random)
    }
}
